ykman/fido.py

```diff
@@ -63,3 +68,67 @@ def reset(self, touch_callback=None):
         finally:
             if (touch_callback):
                 touch_timer.cancel()
+
+    @property
+    def is_fips(self):
+        return False
+
+
+class FipsU2fController(object):
+
+    def __init__(self, driver):
+        self.driver = driver
+        self.ctap = CTAP1(driver._dev)
+
+    @property
+    def has_pin(self):
+        # We don't know, but the change and set commands are the same here.
+        return True
+
+    def set_pin(self, pin):
+        raise NotImplementedError('Use the change_pin method instead.')
+
+    def change_pin(self, old_pin, new_pin):
+        new_length = len(new_pin)
+
+        old_pin = old_pin.encode('utf-8')
```

Comment:
.encode() is fine

Reply:
Decided in F2F discussion to keep this as is.